scripts/gen_configs.py

```diff
@@ -0,0 +1,153 @@
+# TODO pytorch
```

Comment:
This file is part of an effort to automate the generation of configs for use in examples, testing etc. Will likely remove this and save somewhere else for now...